docs/user/security/securing-communications/index.asciidoc

```diff
@@ -3,122 +3,131 @@
 ++++
 <titleabbrev>Encrypting communications</titleabbrev>
 ++++
+Secure Sockets Layer (SSL) and Transport Layer Security (TLS) provide encryption for data-in-transit. While these terms are often used
+interchangeably, {kib} supports only TLS, which supersedes the old SSL protocols.
 
-{kib} supports Transport Layer Security (TLS/SSL) encryption for client 
-requests. 
-//TBD: It is unclear what "client requests" are in this context. Is it just 
-// communication between the browser and the Kibana server or are we talking 
-// about other types of clients connecting to the Kibana server?
+Browsers send traffic to {kib} and {kib} sends traffic to {es}. These communication channels are configured separately to use TLS.
 
-If you are using {security} or a proxy that provides an HTTPS endpoint for {es}, 
-you can configure {kib} to access {es} via HTTPS. Thus, communications between 
-{kib} and {es} are also encrypted. 
+TLS requires X.509 certificates to authenticate the communicating parties and perform encryption of data-in-transit. Each certificate
```

Comment:
nit:
```suggestion
TLS requires X.509 certificates and keys to authenticate the communicating parties and perform encryption of data-in-transit. Each certificate
```

Reply:
@jkakavas the keys are mentioned on the line below -- I don't think it makes sense to mention the keys again here.